spec/features/user_can_log_in_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'helpers/users_helper_spec'
+
+RSpec.feature 'Sign_in', type: :feature do
+  scenario 'Can Login after being signed up with correct email and password' do
+    sign_up_test
+    expect(page).to have_content('Welcome')
+    click_link 'Log out'
+    visit 'sign_in'
+    expect(page).to have_content('Log in')
```

Comment:
Expectation not needed as you use them later on

Reply:
Got rid of them now, used them mainly during writing to debug failing tests, but no longer necessary